### ISSUE TITLE
Mass Obsoletion: focus on branch 'cardiovascular disorder' (MONDO:0004995) 

### ISSUE THREAD

**Issue by sabrinatoro** [monarch-initiative/mondo#6740]
branch 'cardiovascular disorder' (MONDO:0004995) 

Terms to obsolete: 
obsoletion candidate ID | obsoletion candidate label
-- | --
http://purl.obolibrary.org/obo/MONDO_0019826 | abnormal origin or aberrant course of coronary artery
http://purl.obolibrary.org/obo/MONDO_0019816 | anomaly of the tricuspid subvalvular apparatus
http://purl.obolibrary.org/obo/MONDO_0020286 | aortic malformation
http://purl.obolibrary.org/obo/MONDO_0019822 | arterial duct anomaly
http://purl.obolibrary.org/obo/MONDO_0020293 | ascending aorta anomaly
http://purl.obolibrary.org/obo/MONDO_0019837 | atrial appendage anomaly
http://purl.obolibrary.org/obo/MONDO_0020294 | atrial defect and interatrial communication
http://purl.obolibrary.org/obo/MONDO_0020288 | atrioventricular valve anomaly
http://purl.obolibrary.org/obo/MONDO_0008406 | autosomal recessive Emery-Dreifuss muscular dystrophy
http://purl.obolibrary.org/obo/MONDO_0018788 | COL4A1 or COL4A2-related cerebral small vessel disease
http://purl.obolibrary.org/obo/MONDO_0018720 | common cystic lymphatic malformation
http://purl.obolibrary.org/obo/MONDO_0019831 | congenital anomaly of the coronary sinus
http://purl.obolibrary.org/obo/MONDO_0018185 | congenital anomaly of the great veins
http://purl.obolibrary.org/obo/MONDO_0018771 | congenital anomaly of ventricular septum
http://purl.obolibrary.org/obo/MONDO_0017710 | congenital systemic veins anomaly
http://purl.obolibrary.org/obo/MONDO_0020392 | discrete fixed membranous subaortic stenosis
http://purl.obolibrary.org/obo/MONDO_0016899 | Duchenne and Becker muscular dystrophy
http://purl.obolibrary.org/obo/MONDO_0015470 | familial isolated dilated cardiomyopathy
http://purl.obolibrary.org/obo/MONDO_0019150 | familial isolated restrictive cardiomyopathy
http://purl.obolibrary.org/obo/MONDO_0018758 | familial patent arterial duct
http://purl.obolibrary.org/obo/MONDO_0020284 | heart position anomaly
http://purl.obolibrary.org/obo/MONDO_0033947 | hereditary angioedema with normal C1Inh
http://purl.obolibrary.org/obo/MONDO_0016229 | hereditary vascular anomaly
http://purl.obolibrary.org/obo/MONDO_0018831 | HTRA1-related cerebral small vessel disease
http://purl.obolibrary.org/obo/MONDO_0008347 | idiopathic and/or familial pulmonary arterial hypertension
http://purl.obolibrary.org/obo/MONDO_0017129 | inherited cardiac tumor
http://purl.obolibrary.org/obo/MONDO_0018791 | Moyomoya angiopathy
http://purl.obolibrary.org/obo/MONDO_0016348 | non-genetic cardiac rhythm disease
http://purl.obolibrary.org/obo/MONDO_0019541 | non-infectious posterior uveitis
http://purl.obolibrary.org/obo/MONDO_0024471 | non-inflammatory vasculopathy
http://purl.obolibrary.org/obo/MONDO_0015488 | predominantly large-vessel vasculitis
http://purl.obolibrary.org/obo/MONDO_0015489 | predominantly medium-vessel vasculitis
http://purl.obolibrary.org/obo/MONDO_0015490 | predominantly small-vessel vasculitis
http://purl.obolibrary.org/obo/MONDO_0020287 | pulmonary artery or pulmonary branch anomaly
http://purl.obolibrary.org/obo/MONDO_0018640 | secondary vasculitis
http://purl.obolibrary.org/obo/MONDO_0016230 | simple vascular malformation
http://purl.obolibrary.org/obo/MONDO_0020285 | transposition of the great arteries and conotruncal cardiac anomaly
http://purl.obolibrary.org/obo/MONDO_0019063 | vascular anomaly

**Comment by twhetzel** [comment]
I've assigned the orphaned terms. Working on the others.

**Comment by nicolevasilevsky** [comment]
thank you!

**Comment by twhetzel** [comment]
Question - why is `non-infectious posterior uveitis` (MONDO:0019541) being obsoleted? Maintaining the information that these are non-infectious seems useful, e.g. https://eye.hms.harvard.edu/book/what-posterior-uveitis-0.

**Comment by twhetzel** [comment]
The LTB terms are now reviewed and this is not "Ready for Review".

**Comment by nicolevasilevsky** [comment]
@twhetzel- do you mean this is _now_ ready for review? I can review it, thank you!

**Comment by nicolevasilevsky** [comment]
Looking at the spreadsheet, it looks like it is still in progress and not ready for review.

**Comment by nicolevasilevsky** [comment]
@twhetzel I am done reviewing this, please take a look at my comments and let me now if you have any questions

**Comment by sabrinatoro** [comment]
Expert review feedback: https://docs.google.com/spreadsheets/d/1fIqp9B434HjNWZmjqWKYXjvDY6e3LsTg/edit#gid=1997587776

TO DO:
- [ ] add new parents ("ROBOT Reviewer-Curator Agree - Stay" Tab)
- [ ] add new comments ("ROBOT Reviewer-Curator Agree - Leave" Tab)
- [ ] add new parents ("ROBOT Reviewer-Stay Curator-Leave" Tab)
- [ ] review "Reviewer-Leave Curator-Stay" Tab **[TO DO WILL BE UPDATED ACCORDINGLY]**

Note: Once the to-do list is done, 30 terms for which reviewers disagreed will remain to be assessed

**Comment by sabrinatoro** [comment]
There are some comments from the community that should be reviewed, but this is future work. This issue can be closed for now.
